source/material_model/rheology/diffusion_creep.cc

```diff
@@ -99,7 +99,7 @@ namespace aspect
 
         Assert (viscosity_diffusion > 0.0,
                 ExcMessage ("Negative diffusion viscosity detected. This is unphysical and should not happen. "
-                            "Check for negative parameters."));
+                            "Check for negative parameters. Temperature and pressure are " + std::to_string(temperature) + " K, " + std::to_string(pressure) + " Pa. "));
```

Comment:
Sorry @sibiaoliu , instead of using `std::to_string`, could you use `Utilities::to_string`? That seems to be the most used way to convert to a string. Same for the other file.

Reply:
Also, it would be nice if you can split the sentence over two lines, so it's not so long.

Reply:
Sure.